DESCRIPTION

```diff
@@ -20,5 +20,7 @@ Suggests:
     testthat,
     dplyr,
     knitr,
-    rmarkdown
+    rmarkdown,
+    data.table
+FasterWith: data.table
```

Comment:
We can drop the `FasterWith`

Reply:
I recognize this one is not necessary; but I thought it would be handy to signal the nature of the suggestion (testthat, rmarkdown, etc. seem obviously about dev/building the package by definition) -- upon reading the information about DESCRIPTION files it doesn't sound like it's a problem to add arbitrary fields. What does everyone think about this?

Reply:
There's probably a trade-off between "hey, the package devs are trying to tell me something" and "hey, the package devs have created a non-standard field that I need to think about"